.travis.yml

```diff
@@ -18,15 +17,9 @@ env:
     - JOB=test AST_COMPARE=1
 
 matrix:
-  fast_finish: true
```

Comment:
Ref: https://blog.travis-ci.com/2013-11-27-fast-finishing-builds

> With fast finishing enabled, Travis CI will mark your build as finished as soon as one of two conditions are met:
>
> - The only remaining jobs are allowed to fail, or 
> - **a job has already failed.**
> 
> In these cases, the status of the build can already be determined, so there’s no need to wait around until the other jobs finish.

We should leave it enabled.